src/Sarif.Driver/Sdk/SkimmerBase.cs

```diff
@@ -55,12 +57,12 @@ public SkimmerBase()
 
         private Dictionary<string, string> InitializeMessageTemplates()
         {
-            return RuleUtilities.BuildDictionary(ResourceManager, TemplateIds, Id);
+            return RuleUtilities.BuildDictionary(ResourceManager, TemplateResourceIds, ruleId: Id);
```

Comment:
Add the parameter name for clarity, since on first reading its not obvious what this is the "id" of.